content/tutorials/getusermedia/intro/en/index.html

```diff
@@ -130,13 +130,12 @@ <h3 id="toc-round1">Round 1: HTML Media Capture</h3>
 <pre class="prettyprint"><code>&lt;input type="file" accept="image/*;capture=camera"&gt;
 </code></pre>
 
-<p>Pretty nice, right? I particularly like that it reuses a file input. Semantically,
-it makes a lot of sense. Where this particular API falls short is the ability to do real-time effects, such as render live webcam data to a <code>&lt;canvas&gt;</code> and apply WebGL filters.
+<p>Pretty nice, right? Semantically, it makes a lot of sense. Where this particular API falls short is the ability to do real-time effects, such as render live webcam data to a <code>&lt;canvas&gt;</code> and apply WebGL filters.
 HTML Media Capture only allows you to record a media file or take a snapshot in time.</p>
-<p><strong>Support:</strong></p>
+<p><strong>Support</strong></p>
 <ul>
-<li><a href="https://developer.android.com/studio/releases/platforms#3.0</a>—one of the first implementations. Check out <a href="https://davidbcalhoun.com/2011/android-3-0-honeycomb-is-first-to-implement-the-device-api">this video</a> to see it in action.</li>
-<li>Chrome for Android (0.16)</li>
+<li><a href="https://developer.android.com/studio/releases/platforms#3.0</a>—one of the first implementations. Check out <a href="https://davidbcalhoun.com/2011/android-3-0-honeycomb-is-first-to-implement-the-device-api">Watch this video</a> to see it in action.</li>
```

Comment:
Should be code below: 

<li><a href="http://developer.android.com/sdk/android-3.0.html">Android 3.0 browser</a>: one of the first implementations. Check out <a href="https://davidbcalhoun.com/2011/android-3-0-honeycomb-is-first-to-implement-the-device-api">this video</a> to see it in action.</li>

Reply:
Fixed